diff --git a/packages/core/src/global.ts b/packages/core/src/global.ts
index a192a4b..d63c10f 100644
--- a/packages/core/src/global.ts
+++ b/packages/core/src/global.ts
@@ -7,7 +7,7 @@ import { Flock } from "./util/flock"
 import { Flag } from "./flag/flag"
 import { makeGlobalNode } from "./effect/app-node"
 
-const app = "opencode"
+const app = "fylun-code"
 const data = path.join(xdgData!, app)
 const cache = path.join(xdgCache!, app)
 const config = path.join(xdgConfig!, app)
