diff --git a/packages/tui/src/util/presentation.ts b/packages/tui/src/util/presentation.ts
index cf432bf..53e26c3 100644
--- a/packages/tui/src/util/presentation.ts
+++ b/packages/tui/src/util/presentation.ts
@@ -1,5 +1,5 @@
 const logo = {
-  left: ["                   ", "█▀▀█ █▀▀█ █▀▀█ █▀▀▄", "█__█ █__█ █^^^ █__█", "▀▀▀▀ █▀▀▀ ▀▀▀▀ ▀~~▀"],
+  left: ["                        ", "█▀▀▀ █__█ █___ █__█ █▀▀▄", "█▀▀_ ▀▄▄█ █___ █__█ █__█", "▀___ ▄▄▄▀ ▀▀▀▀ ▀▀▀▀ ▀~~▀"],
   right: ["             ▄     ", "█▀▀▀ █▀▀█ █▀▀█ █▀▀█", "█___ █__█ █__█ █^^^", "▀▀▀▀ ▀▀▀▀ ▀▀▀▀ ▀▀▀▀"],
 }
 
